BudgetSquirrel.Api/ResponseModels/BudgetDurationResponse.cs

```diff
@@ -0,0 +1,37 @@
+using System;
+using BudgetSquirrel.Business.BudgetPlanning;
+
+namespace BudgetSquirrel.Api.ResponseModels
+{
+  public class BudgetDurationResponse
+  {
+    public const string DurationTypeMonthlyBookEnded = "MonthlyBookEnded";
+    public const string DurationTypeDaySpan = "DaySpan";
+
+    public Guid Id { get; set; }
+
+    public int? NumberDays { get; set; }
+
+    public int? EndDayOfMonth { get; set; }
+
+    public bool? RolloverEndDateOnSmallMonths { get; set; }
```

Comment:
What does this variable do? If there is a money that is under 31 the budget date will roll over to the next month instead of when it ends?

Reply:
Yeah. So like in February and the end day is 31, if this is true, the end date will be march 1 instead of February 28. But if it's false, then the end date will be february 28. Obviously, if the end day is like 21, then it remains unchanged cause it isn't at the end of the month.